docs/guides/how_to_guides/creating_and_editing_expectations/how_to_edit_an_expectation_suite_using_a_disposable_notebook.rst

```diff
@@ -126,7 +126,7 @@ Steps
 
         1. Generate a notebook.
 
-            To edit a suite called movieratings.ratings you could run the CLI command:
+            To edit a suite called ``movieratings.ratings`` you could run the CLI command:
```

Comment:
@anthonyburdi Same -- should we change the example Expectation Suite name to `movieratings.warning`?..

Reply:
Sure! Changed.